R/fetch-token.R

```diff
@@ -11,7 +11,7 @@ fetch_token <- function(scopes, ...) {
     # TODO(craigcitro): Expose error handling and/or silencing here.
     try({
       token <- f(scopes, ...)
-    })
+    }, silent = TRUE)
     if (!is.null(token)) {
```

Comment:
This is slightly unrelated to this PR, but I'd prefer this style of error handling:

```R
token <- tryCatch(f(scopes, ...), error = function(e) NULL)
```